src/tools/tidy/src/deps.rs

```diff
@@ -228,8 +245,8 @@ const PERMITTED_DEPENDENCIES: &[&str] = &[
     "winapi-i686-pc-windows-gnu",
     "winapi-util",
     "winapi-x86_64-pc-windows-gnu",
-    // this is a false-positive: it's only used by rustfmt, but because it's enabled through a feature, tidy thinks it's used by rustc as well.
     "yansi-term",
+    // this is a false-positive: it's only used by rustfmt, but because it's enabled through a feature, tidy thinks it's used by rustc as well.
```

Comment:
Why this change?

Reply:
Just an oversight, I sorted the list automatically and that moved the comment and I didn't notice.

Reply:
Fixed now.